README.md

```diff
@@ -163,15 +163,13 @@ The same as before the installations can be confirmed by checking the installed
 
 OK! The installation was successful. 🎉 🎆
 
-We also work with Docker ![docker](./images/docker.png),currently we are working on a solution where using Docker will be optional, but until then please continue with the installation procedure.
+We also work with Docker ![docker](./images/docker.png). Currently we are working on a solution where using Docker will be optional, but until then please continue with the installation procedure.
 Run the following command in your console:
 
-        sudo apt install -y docker.io docker-compose
+    sudo apt install -y docker.io docker-compose
 
 Verify that Docker and Docker-Compose are installed by getting their versions:
 
-        sudo docker version
```

Comment:
The commands that check Docker version are removed :scissors:  but the verify Docker version sentence above is left.

Reply:
true, nice catch!! 
I fixed it 😁

Reply:
super, thanks